demisto_sdk/commands/format/update_layout.py

```diff
@@ -301,100 +308,68 @@ def remove_non_existent_fields_container_layout(self):
-                layout_tabs=layout_tabs, content_fields=content_fields
+                layout_tabs=layout_tabs, fields_to_remove=fields_not_in_repo
             )
 
     def remove_non_existent_fields_layout(self):
         """
         Remove non-existent fields from a layout.
         """
-        if not self.id_set_file:
-            logger.warning(
-                f"Skipping formatting of non-existent-fields for {self.source_file} as id_set_path argument is missing"
-            )
-            return
-
-        content_fields = self.get_available_content_fields()
-
-        layout = self.data.get("layout", {})
-        layout_sections = layout.get("sections", [])
-        for section in layout_sections:
-            fields = section.get("fields", [])
-            section["fields"] = self.extract_content_fields(
-                fields=fields, content_fields=content_fields
-            )
-
-        self.remove_non_existent_fields_from_tabs(
-            layout_tabs=layout.get("tabs", []), content_fields=content_fields
-        )
-
-    def get_available_content_fields(self):
-        """
-        Get all the available content indicator/incident fields available + all the built in fields.
-        """
-        return (
-            get_all_incident_and_indicator_fields_from_id_set(
-                self.id_set_file, "layout"
-            )
-            + [field.lower() for field in BUILT_IN_FIELDS]
-            + LAYOUT_AND_MAPPER_BUILT_IN_FIELDS
+        logger.warning(
+            f"Skipping formatting of non-existent-fields for {self.source_file} as this content item is deprecated."
         )
-
-    @staticmethod
-    def extract_content_fields(
-        fields: List[Dict], content_fields: List[str]
-    ) -> List[str]:
-        """
-        Get only incident/indicator fields which are part of the id json file.
-
-        Args:
-            fields (list[dict]): list of fields.
-            content_fields (list[str]): all the available content fields from id json.
-
-        Returns:
-            list[str]: fields which are part of the content items (id json).
-        """
-        return [
-            field  # type: ignore[misc]
-            for field in fields
-            if normalize_field_name(field=field.get("fieldId", "")).lower()
-            not in get_invalid_incident_fields_from_layout(
-                layout_incident_fields=fields, content_fields=content_fields
-            )
-        ]
+        return
 
     def remove_non_existent_fields_from_tabs(
-        self, layout_tabs: list, content_fields: List[str]
+            self, layout_tabs: list, fields_to_remove: List[str]
     ):
         """
         Remove non-existent fields which are not part of the id json from tabs.
 
         Args:
             layout_tabs (list[dict]): list of layout tabs.
-            content_fields (list[str]): all the available content fields from id json.
+            fields_to_remove (list[str]): all the available content fields from id json.
```

Comment:
if `fields_to_remove` is only used for `if x in fields_to_remove`, use a `Set[str]` instead of list.